mmdeploy/pytorch/ops/linear.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from:
+# https://github.com/pytorch/pytorch/blob/9ade03959392e5a90b74261012de1d806cab2253/torch/onnx/symbolic_opset9.py
+
+from mmdeploy.core import SYMBOLIC_REWRITER
+from mmdeploy.utils import Backend
+
+
+@SYMBOLIC_REWRITER.register_symbolic(
+    'linear',
+    is_pytorch=True,
+    arg_descriptors=['v', 'v', 'v', 'f', 'f', 'i', 'i'],
+    backend=Backend.NCNN.value)
+def linear__ncnn(ctx, g, input, weight, bias):
+    """Support export linear This rewrite enable export Gemm."""
+    return g.op(
+        'mmdeploy::Gemm',
```

Comment:
Can we remove `mmdeploy` domain? It looks like the same as the official one [ONNX Gemm](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Gemm-11)